args.go

```diff
@@ -44,28 +45,85 @@ func (cfg *Config) UpgradeDir(upgradeName string) string {
 	return filepath.Join(cfg.Root(), upgradesDir, safeName)
 }
 
+//Symbolinking to genesis
+func (cfg *Config) SymLinkToGenesis() string  {
```

Comment:
Not really sure what this is supposed to do. You can simply take https://github.com/regen-network/cosmosd/blob/master/upgrade.go#L137-L160 and modify it a bit.

Returning `(string, error)` is probably best here, as we may be unable to create the link and must abort with a proper message. This also implies `CurrentBin()` should return `(string, error)` and updates to anything calling it.